src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -138,9 +137,13 @@ ServerHandlerImp::setup(Setup const& setup, beast::Journal journal)
 //------------------------------------------------------------------------------
 
 void
-ServerHandlerImp::onStop()
+ServerHandlerImp::stop()
```

Comment:
Is this function ever invoked in the source base? If not, is it intentional?

Reply:
`Stoppable::onStop` was the first of two stop callbacks in the call tree rooted at `RootStoppable::stop`. It was always called at shutdown.